src/behavior-considered-undefined.md

```diff
@@ -38,6 +38,8 @@ code.
   * A discriminant in an `enum` not included in the type definition.
   * A value in a `char` which is a surrogate or above `char::MAX`.
   * Non-UTF-8 byte sequences in a `str`.
+* Executing code compiled with platform features that the current platform
```

Comment:
```suggestion
  * Executing code compiled with platform features that the current platform
```

Missing leading spaces.

Reply:
I'm not sure I understand. The indented list is of the previous thing ("Invalid values in primitive types").

Reply:
Ah. That's what happens when I don't look at the whole context.